src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.DukeCommandException;
+import duke.task.Task;
+
+public class ListCommand extends Command {
+    public ListCommand(String inputCommand) {
+        super(inputCommand);
+    }
+
+    @Override
+    public void execute(TaskList list, Storage storage, Ui ui) throws DukeCommandException {
+        if (this.getInputCommand().split(" ").length != 1) {
+            throw new DukeCommandException("\u2639 OOPS!!! Wrong 'list' command parameters!");
+        }
+
+        if (list.getList().size() == 0) {
+            ui.printMessage("There is nothing in the list!");
+        } else {
+            ui.printMessage("Here are the tasks in your list:");
+
+            int ctr = 1;
```

Comment:
Maybe can use the long form to make it easier to understand, I am not sure what 'ctr' means from this line